src/components/AllServices/AllServicesGroup.tsx

```diff
@@ -1,15 +1,15 @@
 import { Text, TextVariants } from '@patternfly/react-core/dist/dynamic/components/Text';
 import React, { Fragment } from 'react';
-import { ITLess, titleToId } from '../../utils/common';
+import { titleToId } from '../../utils/common';
 
 import AllServicesLink from './AllServicesLink';
-import { AllServicesGroup, isAllServicesLink } from './allServicesLinks';
+import { AllServicesGroup } from './allServicesLinks';
 
 export type AllServicesGroupProps = AllServicesGroup & {
   category: string;
 };
 const AllServicesGroup = ({ title, links, category }: AllServicesGroupProps) => {
-  const filteredLinks = ITLess() ? links.filter((link) => isAllServicesLink(link) && link.ITLess) : links;
```

Comment:
@wcater803 I'm not sure we want to remove the `isAllServicesLink(link)` check here - just the `&& link.ITLess` portion? What do you think?

Reply:
Chatted with @wcater803  - I missed the left hand part of the expression (`ITLess()`). My bad.